MultiTable/src/template/modal-sample.handlebars

```diff
@@ -29,7 +29,7 @@
             </p>
             <h5>{{heading.message}}</h5>
             <p>
-              {{data.messageCode}}{{#if data.messageBody}}{{#if data.messageCode}}:{{/if}}
+                {{data.messageCode}}{{#if data.messageCode}}{{#if data.aspect.valueLabel}} {{data.aspect.valueLabel}}{{/if}}{{/if}}{{#if data.messageBody}}{{#if data.messageCode}}:{{/if}}
```

Comment:
Should this have parentheses?

Reply:
If valueType is `mbps` and messageCode is `170` then I would want it to read `170 mbps: blah blah blah messageBody blah blah`.